x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/get_baseline_and_deviation_rates.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { LOG_RATE_ANALYSIS_TYPE } from '@kbn/aiops-log-rate-analysis';
+
+export function getLogRateChange(
+  analysisType: typeof LOG_RATE_ANALYSIS_TYPE[keyof typeof LOG_RATE_ANALYSIS_TYPE],
+  baselineBucketRate: number,
+  deviationBucketRate: number
+) {
+  const logRateChange =
+    baselineBucketRate > 0
+      ? analysisType === LOG_RATE_ANALYSIS_TYPE.SPIKE
```

Comment:
Our style guide recommends not doing nested ternaries. I'd suggest just using if/else in this function and return early if something matches.

Reply:
Updated in ff861eb3f280d3812b7536fcbabcd06eced039cc